packages/validator/src/cli-validator/run-validator.js

```diff
@@ -196,15 +196,6 @@ async function runValidator(cliArgs, parseOptions = {}) {
       logError('There was a problem with spectral.', getError(err));
       logger.error('Additional error details:');
       logger.error(err);
-      if (
-        err.message ==
-        "Cannot use 'in' operator to search for '**' in undefined"
-      ) {
-        logger.error(
-          'This error likely means the API Definition is missing a `servers` field.\n' +
-            'Spectral currently has a bug that prevents it from processing a definition without a `servers` field.'
-        );
-      }
```

Comment:
Good catch - did you confirm that this is no longer an issue?

Reply:
yes, I meant to add a comment here about this...  I verified that Spectral doesn't keel over if "servers" is missing.  Instead, there is now apparently a rule about it because it instead displayed a validation error about "servers" missing.

Reply:
That's great!